src/Symfony/Component/ObjectMapper/TargetClass.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\ObjectMapper;
```

Comment:
The classes implementing `ConditionCallableInterface` can be in a `Condition` subnamespace.